src/textual/widgets/_header.py

```diff
@@ -160,12 +160,28 @@ def watch_tall(self, tall: bool) -> None:
     def _on_click(self):
         self.toggle_class("-tall")
 
+    @property
+    def screen_title(self) -> str:
```

Comment:
Docstrings pls.

Could we make these computed reactives? That way you could watch `self.screen_title`

Reply:
See 5a15e9c8a for docstrings.
As per our conversation, we can't make them computed reactives because we can't trigger the compute method when the reactives on `Screen` or `App` change.